require-config.js

```diff
@@ -4,6 +4,7 @@ require.config({
     'chai': 'node_modules/chai/chai',
     'sinon': 'node_modules/sinon/pkg/sinon',
     'sinon-chai': 'node_modules/sinon-chai/lib/sinon-chai',
-    'knockout': 'node_modules/knockout/build/output/knockout-latest'
+    'knockout': 'node_modules/knockout/build/output/knockout-latest',
+    'text': 'node_modules/requirejs-text/text'
```

Comment:
Added the text plugin to the require JS config so that require JS knows where to find the plugin